### PR TITLE
Some entity attribute docs

### PR DESCRIPTION
**Pull request by i509VCB**

### CODE REVIEW

File: mappings/net/minecraft/entity/attribute/ClampedEntityAttribute.mapping [hunk 1]
```diff
@@ -1,4 +1,5 @@
 CLASS net/minecraft/class_1329 net/minecraft/entity/attribute/ClampedEntityAttribute
+	COMMENT Represents an attribute which limits the minimum and maximum value of an attribute instance.
```
Comment (haykam821) [inline mappings/net/minecraft/entity/attribute/ClampedEntityAttribute.mapping:2]:
```suggestion
	COMMENT Represents an attribute which limits the minimum and maximum value of an attribute instance.
```

File: mappings/net/minecraft/entity/attribute/EntityAttribute.mapping [hunk 2]
```diff
@@ -1,14 +1,20 @@
 CLASS net/minecraft/class_1320 net/minecraft/entity/attribute/EntityAttribute
+	COMMENT Represents an attribute that a living entity may have.
+	COMMENT
+	COMMENT <p>An attrbitute is a tracked value stored on an entity.
```
Comment (haykam821) [inline mappings/net/minecraft/entity/attribute/EntityAttribute.mapping]:
```suggestion
	COMMENT <p>An attribute is a tracked value stored on an entity.
```

**Review by haykam821 — CHANGES_REQUESTED** [review]

File: mappings/net/minecraft/entity/attribute/ClampedEntityAttribute.mapping (hunk 1 above)
Comment (liach) [inline mappings/net/minecraft/entity/attribute/ClampedEntityAttribute.mapping]:
```suggestion
	COMMENT Represents an attribute with minimum and maximum value limits.
```

File: mappings/net/minecraft/entity/attribute/DefaultAttributeContainer.mapping [hunk 3]
```diff
@@ -15,6 +15,11 @@ CLASS net/minecraft/class_5132 net/minecraft/entity/attribute/DefaultAttributeCo
 	METHOD method_27308 getModifierValue (Lnet/minecraft/class_1320;Ljava/util/UUID;)D
 		ARG 1 attribute
 		ARG 2 uuid
+	METHOD method_27309 containsModifier (Lnet/minecraft/class_1320;Ljava/util/UUID;)Z
+		ARG 1 type
+		ARG 2 uuid
+	METHOD method_27310 contains (Lnet/minecraft/class_1320;)Z
```
Comment (liach) [inline mappings/net/minecraft/entity/attribute/DefaultAttributeContainer.mapping]: `has`/`hasModifier`

File: mappings/net/minecraft/entity/attribute/EntityAttribute.mapping [hunk 4]
```diff
@@ -1,14 +1,20 @@
 CLASS net/minecraft/class_1320 net/minecraft/entity/attribute/EntityAttribute
+	COMMENT Represents an attribute that a living entity may have.
```
Comment (liach) [inline mappings/net/minecraft/entity/attribute/EntityAttribute.mapping]:
```suggestion
	COMMENT Represents a type of double-valued attribute that a living entity may have.
```

File: mappings/net/minecraft/entity/attribute/EntityAttributeInstance.mapping [hunk 5]
```diff
@@ -1,7 +1,8 @@
 CLASS net/minecraft/class_1324 net/minecraft/entity/attribute/EntityAttributeInstance
+	COMMENT Represents an instance of a type of attribute.
 	FIELD field_23700 type Lnet/minecraft/class_1320;
 	FIELD field_23701 operationToModifiers Ljava/util/Map;
-	FIELD field_23702 byId Ljava/util/Map;
+	FIELD field_23702 modifiersById Ljava/util/Map;
```
Comment (liach) [inline mappings/net/minecraft/entity/attribute/EntityAttributeInstance.mapping]:
```suggestion
	FIELD field_23702 idToModifiers Ljava/util/Map;
```

File: mappings/net/minecraft/entity/attribute/EntityAttributeInstance.mapping [hunk 6]
```diff
@@ -1,7 +1,8 @@
 CLASS net/minecraft/class_1324 net/minecraft/entity/attribute/EntityAttributeInstance
+	COMMENT Represents an instance of a type of attribute.
```
Comment (liach) [inline mappings/net/minecraft/entity/attribute/EntityAttributeInstance.mapping]:
```suggestion
	COMMENT A double-valued attribute.
```

File: mappings/net/minecraft/entity/attribute/EntityAttributeInstance.mapping [hunk 7]
```diff
@@ -11,17 +12,35 @@ CLASS net/minecraft/class_1324 net/minecraft/entity/attribute/EntityAttributeIns
...
+		COMMENT The modifier will not be serialized.
 		ARG 1 modifier
+	METHOD method_26836 (Lnet/minecraft/class_1322$class_1323;)Ljava/util/Set;
+		ARG 0 operation
 	METHOD method_26837 addPersistentModifier (Lnet/minecraft/class_1322;)V
 		ARG 1 modifier
 	METHOD method_26838 onUpdate ()V
 	METHOD method_26839 toTag ()Lnet/minecraft/class_2487;
 	METHOD method_26840 computeValue ()D
+		COMMENT Computes the value of an attribute while taking attribute modifiers into account.
```
Comment (liach) [inline mappings/net/minecraft/entity/attribute/EntityAttributeInstance.mapping]:
```suggestion
		COMMENT Computes this attribute's value, taking modifiers into account.
```

File: mappings/net/minecraft/entity/attribute/EntityAttributeModifier.mapping [hunk 8]
```diff
@@ -28,7 +28,16 @@ CLASS net/minecraft/class_1322 net/minecraft/entity/attribute/EntityAttributeMod
 	METHOD method_6186 getValue ()D
 	METHOD method_6189 getId ()Ljava/util/UUID;
 	CLASS class_1323 Operation
+		COMMENT Represents an operation which can be applied to an attribute modifier.
+		FIELD field_6328 ADDITION Lnet/minecraft/class_1322$class_1323;
+			COMMENT Adds to the base value of an attribute.
 		FIELD field_6329 id I
+		FIELD field_6330 MULTIPLY_BASE Lnet/minecraft/class_1322$class_1323;
+			COMMENT Multiplies the base value of the attribute.
```
Comment (liach) [inline mappings/net/minecraft/entity/attribute/EntityAttributeModifier.mapping:36]: does multiply base go before addition? if yes i'd prefer to specify that

Comment (i509VCB) [reply mappings/net/minecraft/entity/attribute/EntityAttributeModifier.mapping:36]: Multiply base occurs after addition, but uses the base value.

File: mappings/net/minecraft/entity/attribute/DefaultAttributeContainer.mapping (hunk 3 above)
Comment (liach) [reply mappings/net/minecraft/entity/attribute/DefaultAttributeContainer.mapping]: These are new names and are safe to rename.
```suggestion
	METHOD method_27310 has (Lnet/minecraft/class_1320;)Z
```

File: mappings/net/minecraft/entity/attribute/DefaultAttributeContainer.mapping [hunk 9]
```diff
@@ -15,6 +15,11 @@ CLASS net/minecraft/class_5132 net/minecraft/entity/attribute/DefaultAttributeCo
 	METHOD method_27308 getModifierValue (Lnet/minecraft/class_1320;Ljava/util/UUID;)D
 		ARG 1 attribute
 		ARG 2 uuid
+	METHOD method_27309 containsModifier (Lnet/minecraft/class_1320;Ljava/util/UUID;)Z
```
Comment (liach) [inline mappings/net/minecraft/entity/attribute/DefaultAttributeContainer.mapping]:
```suggestion
	METHOD method_27309 hasModifier (Lnet/minecraft/class_1320;Ljava/util/UUID;)Z
```

File: mappings/net/minecraft/entity/attribute/ClampedEntityAttribute.mapping [hunk 10]
```diff
@@ -1,4 +1,5 @@
 CLASS net/minecraft/class_1329 net/minecraft/entity/attribute/ClampedEntityAttribute
+	COMMENT Represents an attribute with minimum and maximum value limits.
```
Comment (liach) [inline mappings/net/minecraft/entity/attribute/ClampedEntityAttribute.mapping]:
```suggestion
	COMMENT Represents a type of attribute with minimum and maximum value limits.
```
Should have fixed that earlier. `Attribute` apparently are types than instances.

File: mappings/net/minecraft/entity/attribute/EntityAttribute.mapping [hunk 11]
```diff
@@ -1,14 +1,20 @@
...
 		ARG 1 translationKey
 		ARG 2 fallback
 	METHOD method_26829 setTracked (Z)Lnet/minecraft/class_1320;
+		COMMENT Sets all instances of this attribute to sync their values to the client.
 		ARG 1 tracked
 	METHOD method_26830 getTranslationKey ()Ljava/lang/String;
 	METHOD method_6165 clamp (D)D
 		ARG 1 value
 	METHOD method_6168 isTracked ()Z
+		COMMENT Checks if instances of this attribute should sync values to the client.
```
Comment (liach) [inline mappings/net/minecraft/entity/attribute/EntityAttribute.mapping]:
```suggestion
		COMMENT Checks if instances of this attribute should synchronize values to clients.
```

File: mappings/net/minecraft/entity/attribute/EntityAttribute.mapping [hunk 12]
```diff
@@ -1,14 +1,20 @@
...
+	COMMENT <p>An attribute is a tracked value stored on an entity.
+	COMMENT An attribute has a default value, which can be changed via use of attribute modifiers.
 	FIELD field_23697 fallback D
 	FIELD field_23698 tracked Z
 	FIELD field_23699 translationKey Ljava/lang/String;
 	METHOD <init> (Ljava/lang/String;D)V
 		ARG 1 translationKey
 		ARG 2 fallback
 	METHOD method_26829 setTracked (Z)Lnet/minecraft/class_1320;
+		COMMENT Sets all instances of this attribute to sync their values to the client.
```
Comment (liach) [inline mappings/net/minecraft/entity/attribute/EntityAttribute.mapping]:
```suggestion
		COMMENT Sets all instances of this attribute to synchronize their values to clients.
```

File: mappings/net/minecraft/entity/attribute/EntityAttribute.mapping [hunk 13]
```diff
@@ -1,14 +1,20 @@
 CLASS net/minecraft/class_1320 net/minecraft/entity/attribute/EntityAttribute
+	COMMENT Represents a type of double-valued attribute that a living entity may have.
+	COMMENT
+	COMMENT <p>An attribute is a tracked value stored on an entity.
```
Comment (liach) [inline mappings/net/minecraft/entity/attribute/EntityAttribute.mapping]:
```suggestion
	COMMENT <p>An attribute is a tracked double value stored on an entity.
```

File: mappings/net/minecraft/entity/attribute/EntityAttribute.mapping [hunk 14]
```diff
@@ -1,14 +1,20 @@
...
+	COMMENT An attribute has a default value, which can be changed via use of attribute modifiers.
```
Comment (liach) [inline mappings/net/minecraft/entity/attribute/EntityAttribute.mapping]:
```suggestion
	COMMENT An attribute has a default value on which attribute modifiers operate.
```

File: mappings/net/minecraft/entity/attribute/EntityAttributeModifier.mapping [hunk 15]
```diff
@@ -28,7 +28,18 @@ CLASS net/minecraft/class_1322 net/minecraft/entity/attribute/EntityAttributeMod
...
+			COMMENT
+			COMMENT <p>Is applied after addition.
+		FIELD field_6331 MULTIPLY_TOTAL Lnet/minecraft/class_1322$class_1323;
+			COMMENT Multiplies the total value of the attribute.
+			COMMENT
+			COMMENT <p>The total value is the equal to the sum of all additions and base multiplications applied by an attribute modifier.
```
Comment (YanisBft) [inline mappings/net/minecraft/entity/attribute/EntityAttributeModifier.mapping]:
```suggestion
			COMMENT <p>The total value is equal to the sum of all additions and base multiplications applied by an attribute modifier.
```

**Review by YanisBft — APPROVED** [review]

**Review by shartte — APPROVED** [review]

**Review by liach — APPROVED** [review]

**Review by haykam821 — APPROVED** [review]

**Review by Gegy — APPROVED** [review]

**Pull request merged**